ale_linters/go/gometalinter.vim

```diff
@@ -3,6 +3,7 @@
 
 call ale#Set('go_gometalinter_options', '')
 call ale#Set('go_gometalinter_executable', 'gometalinter')
+call ale#Set('go_gometalinter_lint_dir', 1)
```

Comment:
Maybe a better name would be `go_gometalinter_package`? In then in the doc's you can describe how settings this option will lint the whole package instead of the file your currently working on.